bool valid
string camera_name
DetectedFrame[] frames
